### PR TITLE
Fix duplicate code and lint import

### PR DESCRIPTION
**Pull request by lugondev**
Using the JetBrains IDE, I fixed a few warnings.

### CODE REVIEW

**Comment by lugondev** [comment]
@casey give me feedback. thanks

File: src/index/updater.rs [hunk 1]
```diff
@@ -57,15 +57,20 @@ impl<'index> Updater<'_> {
...
+        .insert(
+          height,
+          &SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
+            .map(|duration| duration.as_millis())
+            .unwrap_or(0),
+        )
+        .expect("insert height failed");
+    };
+    write_height_to_ts(&mut wtx, &self.height);
```
Comment (casey) [inline src/index/updater.rs:73]: Can you revert these?

File: src/index/updater.rs [hunk 2]
```diff
@@ -128,15 +133,7 @@ impl<'index> Updater<'_> {
...
-        wtx
-          .open_table(WRITE_TRANSACTION_STARTING_BLOCK_COUNT_TO_TIMESTAMP)?
-          .insert(
-            &self.height,
-            &SystemTime::now()
-              .duration_since(SystemTime::UNIX_EPOCH)
-              .map(|duration| duration.as_millis())
-              .unwrap_or(0),
-          )?;
+        write_height_to_ts(&mut wtx, &self.height);
```
Comment (casey) [inline src/index/updater.rs:136]: Can you revert this change? What was the lint error here?

File: src/object.rs [hunk 3]
```diff
@@ -1,4 +1,5 @@
 use super::*;
```
Comment (casey) [inline src/object.rs:1]: Let's combine these use statements.

**Review by casey — CHANGES_REQUESTED** [review]
Nice, mostly looks good! Just a couple things I would revert, see comments.

File: src/index/updater.rs (hunk 1 above)
Comment (lugondev) [reply src/index/updater.rs:73]: ok

Comment (lugondev) [reply src/index/updater.rs:73]: I see
``` wtx
        .open_table(WRITE_TRANSACTION_STARTING_BLOCK_COUNT_TO_TIMESTAMP)
        .expect("open table failed")
        .insert(
          height,
          &SystemTime::now()
            .duration_since(SystemTime::UNIX_EPOCH)
            .map(|duration| duration.as_millis())
            .unwrap_or(0),
        )``` 
        is duplicate. I will revert these.

File: src/index/updater.rs (hunk 2 above)
Comment (lugondev) [reply src/index/updater.rs:136]: LugonShot 2024-02-17 at 13 19 58@2x

**Comment by lugondev** [comment]
@casey thanks for yours reviews

**Comment by lugondev** [comment]
@casey please review this PR, thankssssss

**Review by raphjaph — APPROVED** [review]
LGTM!

**Review by casey — APPROVED** [review]
LGTM!
